src/api/useMeasureServiceApi.tsx

```diff
@@ -434,7 +434,11 @@ export class MeasureServiceApi {
         signal,
       }
     );
-    return response.data;
+
+    return {
+      status: response.status,
+      data: response.data,
```

Comment:
I think we can just return response object, and use the Promise<AxiosResponse> as a return type